bioagents/msa/msa_module.py

```diff
@@ -245,7 +245,7 @@ def respond_confirm_relation_from_literature(self, content):
             # TODO: Handle this more gracefully, if possible.
             return self.make_failure('MISSING_MECHANISM')
         num_stmts = len(stmts)
-        self.say(finder.describe())
```

Comment:
I'm not sure that we need to turn off include_negative in this case and the one above since the BA, for now, will only generate its own response if we send MISSING_MECHANISM. Here we send SUCCESS instead so I think we'd want to still generate our own response along with that.

Reply:
The BA should be able to, and thus should, generate its own response in the case that there are not statements returned to indicate that no statements were found, in which case sending our own message would be a duplicate, right?

Reply:
Okay, I agree with _should_ but it currently doesn't. It only does for MISSING_MECHANISM. Therefore unless we request further changes in the BA, it won't generate a response in these cases.